### ISSUE TITLE
Add "lipid" keyword to select_atoms

### ISSUE THREAD

**Issue by mimischi** [MDAnalysis/mdanalysis#2082]
**Is your feature request related to a problem? Please describe.**
It would be great if one could select all lipids in a Universe by a common keyword, e.g. `lipid` just like one can select all proteins via `protein`.

**Describe the solution you'd like**
```python
u = mda.Universe(gro)
u.select_atoms("lipid")
```

**Additional context**
VMD does have a similar, partially functional (does not recognize all lipid names) version of this.

I would like to contribute this in a PR. I guess duplicating the behavior of `ProteinSelection` would be the solution?
https://github.com/MDAnalysis/mdanalysis/blob/55e747504e51f873dfffdf8356ce425cdfbc6f08/package/MDAnalysis/core/selection.py#L763-L802

I think grabbing the residue names from charmm-gui.org (see [here](http://charmm-gui.org/?doc=input/membrane), click "Membrane only system") should be sufficient. They include pretty much all parameterized lipids for CHARMM36. Although I do not know how one would need to do attributions here.

**Comment by mimischi** [comment]
Oh, seems like #104 is related.

**Comment by richardjgowers** [comment]
@mimischi yeah sounds like it might be useful, but I can imagine a lot of reports of "it didn't select *my* lipids though" (how universal are the charmm36 names?).  Managing this will require #104.

The way selections work, you can define the class anywhere and it gets automagically included in to the selection language, so you can play around in a notebook and get it working quickly.  PR welcome!

**Comment by mimischi** [comment]
Awesome, will do!

I will try to do some research how lipids are named in other force fields, if no one else knows this already :)
